openedx/adg/lms/applications/managers.py

```diff
@@ -29,6 +31,56 @@ def prereq_course_groups(self):
         """
         return self.get_queryset().filter(is_program_prerequisite=True, multilingual_courses__isnull=False).distinct()
 
+    def get_user_program_prereq_courses(self, user):
+        """
+        Returns program prerequisite courses for a user.
+        Args:
+            user (User): User for which program prerequisite courses will be returned
+        Returns:
+            list: List of program prerequisite courses for a user
+        """
```

Comment:
Lets follow the convention and add spaces between function definition, Args and Returns